vite.config.ts

```diff
@@ -33,6 +33,21 @@ export default defineConfig({
     },
   },
   build: {
-    target: "ES2022"
-  },
+    minify: false,
+    sourcemap: true, // NOTE: sourcemaps may be incorrect, double check!
```

Comment:
Should we remove this note or should I double-check?

Reply:
Usually sourcemaps works well and should make debugging easier, but sometimes sourcemaps can be corrupted by source code transformers (like typia and babel).

At first glance - the source maps CURRENTLY are correct (even for typia), but if you see that the source maps are incorrect, you should disable this or fix them.